app/src/main/java/com/courseproject/tindar/entities/AccountController.java

```diff
@@ -0,0 +1,237 @@
+package com.courseproject.tindar.entities;
+
+import android.location.Location;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+public interface AccountController {
```

Comment:
A controller is something that interacts with frontend, and is not used as an interface for the entity class. Please refer to the UserLoginCleanArchitecture repo. You can maybe use factory design similarly with that repo?